website/docs/r/guardduty_filter.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "aws"
```

Comment:
Our current documentation standards have added the frontmatter entry `subcategory: "GuardDuty"`